### PR TITLE
v!say

### PR DESCRIPTION
**Pull request by emilianya**
Hi, So I made v!say

Fvn said vuk wanted it to check for owner of guild after I made it for if owner of VukkyBot.
So I made both.

Its mostly based on Vukky code.

Config needs to have the owner id as string because discord ids are strings

### CODE REVIEW

File: commands/say.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        //if(message.author.id != message.guild.owner.id) {
+        //message.channel.send(embeds.errorEmbed("Sorry, but you're not the owner of this server."))
+    //} else {
+        //let say = args.slice(0).join(' ')
+        //message.delete()
+        //message.channel.send(say)
+    //}
+
+    //If you want it to check for if you are the owner of the VukkyBot
+
```
Comment (nyakase) [inline commands/say.js]:
```suggestion
```
don't mind me, just making changes in the worst way possible

**Review by nyakase — CHANGES_REQUESTED** [review]
You turned the `owner` in the config into a string without updating the config.schema file respectively
Please fix or I am going to get the gun

File: commands/say.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    if(message.author.id != config.misc.owner) {
+        message.channel.send(embeds.errorEmbed("Sorry, but you're not the owner of this VukkyBot."))
+    } else {
+        let say = args.slice(0).join(' ')
+        message.delete()
+        message.channel.send(say)
+    }
+    },
+};
```
Comment (nyakase) [inline commands/say.js]:
```suggestion
const embeds = require("../embeds.js");
const config = require('../config.json');

module.exports = {
    name: 'say',
    description: 'Make VukkyBot say things!',
    dcPermissions: ['EMBED_LINKS', 'MANAGE_MESSAGES'],
    execute(message, args) {
    if(message.author.id != config.misc.owner) {
        message.channel.send(embeds.errorEmbed("Sorry, but you're not the owner of this VukkyBot."))
    } else {
        let say = args.slice(0).join(' ')
        message.delete()
        message.channel.send(say)
    }
    },
};
```

**Review by nyakase — COMMENTED** [review]
what is happen

**Comment by emilianya** [comment]
It is fix

**Comment by nyakase** [comment]
@all-contributors please add @viluzi for code

**Pull request merged**